### PR TITLE
policy update

### PR DESCRIPTION
**Pull request by realrasengan**

### CODE REVIEW

**Comment by sonOfRa** [comment]
Is this the kind of unannounced policy change Freenode users should generally expect now? You see a behaviour you dislike, announce that it is against Freenode policy, and then retroactively change Freenode policy in order to make that statement true?

**Comment by dhjw** [comment]
> Is this the kind of unannounced policy change Freenode users should generally expect now? You see a behaviour you dislike, announce that it is against Freenode policy, and then retroactively change Freenode policy in order to make that statement true?

Pretty much every site works that way and often changes its terms & conditions.

File: content/pages/policies.md [hunk 1]
```diff
@@ -81,10 +84,11 @@ applicable where freenode operates. Use of freenode and its facilities is a
 privilege, not a right, and may be withdrawn where we believe that this has not
 been the case.
 
-Unlawful activities and related support activities are considered off-topic, as are
-inappropriate advertising, heavy media file trading, proprietary game software modding,
-warez, porn and various forms of antisocial behaviour, including (but not limited to) 
-political, racial, ethnic, religious or gender-related invective.
```
Comment (hanetzer) [inline content/pages/policies.md:87]: Glad to know I can now use political, racial, ethnic, religious or gender-related invective on freenode, this is a move in the right direction!

Comment (dhjw) [reply content/pages/policies.md:87]: Why shouldn't people be allowed to talk about those things or fear being arbitrarily suppressed if they do?

Comment (joepie91) [reply content/pages/policies.md:87]: That is indeed the view generally taken by those with a loud voice, who are not under constant threat.

**Comment by Superfreeze-automated** [comment]
> If you actually read the change you see it provides a benefit to users in one part and in the other it removes rather than imposes limits on conversations.

I would disagree, but it's not obvious why from the rule itself. Esp., read:

> For abandoned project channels that have moved or are no longer available to the public, you may request the ownership of the channel be transferred to you.

Nothing against such a rule in general, but the problem is the context:
- This rule was applied _before_ it was merged here, i.e. the channel ##hntop was taken over (source: https://www.devever.net/~hl/freenode_abuse) and _then_ the Freenode policy was retroactively changed. Also, the rule change was announced or discussed nowhere.
- Timing and this story suggest that the intention of the change is to keep people from leaving Freenode. If a project moves and e.g. sets the topic to "Moved to Libera.Chat", then they can take over the channel (I mean, it's abandoned) and revert the topic.

I believe that Andrew Lee (alias rasengan) is acting in good faith (also, Hanlon's razor, benefit of doubt and so on). And I understand that he doesn't want abandoned channels polluting the namespace. But, to use a service, I would like to be able trust in the operator's decisions, which I can't anymore here.

**Comment by realrasengan** [comment]
Please comment on issue 515. Thank you.

**Pull request merged**